app/views/people/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<% zero_padded_id = @person.id.to_s %>
+<% while zero_padded_id.length < 3 %>
+  <% zero_padded_id.insert(0, "0") %>
+<% end %>
```

Comment:
a little bit obscure, but there is a built in ruby method to do exactly this: http://ruby-doc.org/core-2.2.2/Kernel.html#method-i-sprintf

So this could simply be: `sprintf('%03d', @person.id)`